ClearURLs for uBo/compile.py

```diff
@@ -0,0 +1,198 @@
+# Original license:
+# Copyright © 2021 rusty-snake
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# this code has been modified by https://github.com/iam-py-test, as to improve compatibility and fix issues
+
+import json
+import sys
+from datetime import date
+from typing import IO
+
+import requests  # type: ignore
+
+HEAD = """\
+! Title: ClearURLs for uBo - List extension
+! Homepage: https://github.com/DandelionSprout/adfilt/discussions/163
+! Last updated: {date}
+! Expires: 1 day
+! Note: This was based off of https://gist.github.com/rusty-snake/5cd83a87d680ecbd03e79a1a06758207, which is based off of https://github.com/ClearURLs/Rules. The maintainers of Adfilt (DandelionSprout and iam-py-test, and contributors) have made some modifications as to keep it up-to-date with the source and to fix issues
+
+"""
+KNOWN_BAD_FILTERS = [
+    "$removeparam=/^ref_?=/",
+]
```

Comment:
```suggestion
KNOWN_BAD_FILTERS = [
    "$removeparam=/^ref_?=/",
    "$removeparam=sa,domain=google.*",
    "$removeparam=usg,domain=google.*",
]
```

Reply:
I was doing that right as you commented this